app/views/brexit_checker/_results_citizen_actions.html.erb

```diff
@@ -0,0 +1,31 @@
+<% heading = t('brexit_checker.results.audiences.citizen.heading') %>
```

Comment:
not sure if this variable adds anything. You can just use the translation when necessary

Reply:
see above